src/pychart/_interpreter/statement.py

```diff
@@ -81,3 +81,34 @@ def __call__(self, environment: Environment):
             self.if_body(environment)
         elif self.else_body and not test_result:
             self.else_body(environment)
+
+
+class pychBreakStatementExceptionJeffer(Exception):
+    pass
+
+class Break(Stmt):
+    def __call__(self, environment: Environment):
+        raise pychBreakStatementExceptionJeffer("The jeeffer did not correspond")
+
```

Comment:
For readability and better error messages for the user

```suggestion
class BreakStatementException(Exception):
    pass

class Break(Stmt):
    def __call__(self, environment: Environment):
        raise BreakStatementException("Cannot invoke 'break;' outside of a while loop")

```